docs/BUILD

```diff
@@ -0,0 +1,95 @@
+load("@io_bazel_skydoc//stardoc:stardoc.bzl",
+     "stardoc")
+
+# Note that due to a bug in Bazel 0.22.0, these targets cannot be built without
+# the --experimental_remap_main_repo flag.
+stardoc(
+    name = "build_test_docs",
+    input = "//rules:build_test.bzl",
+    out = "build_test_doc_gen.md",
+    deps = ["//rules:build_test"],
+)
```

Comment:
This is very verbose.
Should there be a macro to help?

Reply:
I don't think it's that verbose, since the verbosity is referring to explicit targets and files. I'd rather leave this as-is for now.